contracts/babylon/src/contract.rs

```diff
@@ -309,15 +311,109 @@ pub fn execute(
             // TODO: Add events (#124)
             Ok(res)
         }
+        ExecuteMsg::DistributeRewards { fp_distribution } => {
```

Comment:
How about `SendRewardsDistribution` or something similar to better clarify that it does not perform the distribution but forwards the distribution info to Babylon Genesis?